incubator/hydra/templates/deployment.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ include "hydra.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "hydra.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ include "hydra.chart" . }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "hydra.name" . }}
+  template:
+    metadata:
+      name: {{ include "hydra.fullname" . }}
+      labels:
+        app.kubernetes.io/name: {{ include "hydra.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/managed-by: {{ .Release.Service }}
+        helm.sh/chart: {{ include "hydra.chart" . }}
+    spec:
+{{- if .Values.postgresql.enabled }}
+      initContainers:
+      - name: db
+        image: {{ printf "%s:%s" .Values.image .Values.imageTag | quote }}
+        imagePullPolicy: {{ .Values.imagePullPolicy | quote }}
+        args:
+        - migrate 
+        - sql
+        - -e
+        env:
+{{ include "database" . | indent 8 }}
+{{ include "settings" . | indent 8 }}
+{{- end }}
+      containers:
+      - name: hydra
+        image: {{ printf "%s:%s" .Values.image .Values.imageTag | quote }}
+        imagePullPolicy: {{ .Values.imagePullPolicy | quote }}
+        args:
+        - serve
```

Comment:
I have tried this on an image compiled from the current master, try setting the flags like this:
```
- --config /hydra/config.yaml
- serve
- all
```

Reply:
Updated!